cmd/bootstrap/run/qc.go

```diff
@@ -49,6 +49,15 @@ func (pd *ParticipantData) Identities() flow.IdentityList {
 	return bootstrap.ToIdentityList(nodes)
 }
 
+// DKGIndexMap returns a map from node ID to DKG index. Can be used to reconstruct EpochCommit.
+func (pd *ParticipantData) DKGIndexMap() map[flow.Identifier]int {
```

Comment:
following up on [my previous comment](https://github.com/onflow/flow-go/pull/6338#discussion_r1735304862), I think it would improve code safety if we used a dedicated type for the returned map, that clearly documents the protocol-mandated invariants.

Reply:
[`1a0c5be` (#6338)](https://github.com/onflow/flow-go/pull/6338/commits/1a0c5bedadf8d840d146c60b27eed4ed4f06d7fb)